### PR TITLE
[Rules] Add templates

### PR DESCRIPTION
**Pull request by pedrosousa**
### Summary

Addresses PCX-11615.
Adds references to the new **Rules** > **Templates** page.

### Documentation checklist

- [X] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/changelogs/rules.yaml [hunk 1]
```diff
@@ -5,66 +5,70 @@ productLink: "/rules/"
 productArea: Application performance
 productAreaLink: /fundamentals/reference/changelog/performance/
 entries:
+  - publish_date: "2024-09-05"
```
Comment (pedrosousa) [inline src/content/changelogs/rules.yaml:8]: I just added this new entry.

**Review by kodster28 — APPROVED** [review]

**Review by ijunxyz123 — COMMENTED** [review]
pedro/2024-09-04-rules-templates

**Review by ijunxyz123 — COMMENTED** [review]
pedro/2024-09-04-rules-templates

**Comment by ijunxyz123** [comment]
.patch

File: src/content/changelogs/rules.yaml (hunk 1 above)
Comment (ijunxyz123) [reply src/content/changelogs/rules.yaml:8]: pedro/2024-09-04-rules-templates

**Review by ijunxyz123 — COMMENTED** [review]
from tqdm import trange      # visualizing the progress
from typing import Callabe

# define a simple decorator(inner function) to mesure the execution of a function
def measure_time(func:Callable)->Callable:
    '''
    args: one argument, is the function that we want to decorated(e,g functions that:
    build the model, train the model, optimize the model, ...etc)
    '''
    # define the wrapper function that executes the original function(e,g arg func)
    def wrapper(*args, **kwargs)->float:
        '''
        the wrapper function have the same argument as the object(e,g arg)function
        '''
        start_time=time()
        # call the object function to process and manipulate ...
        result=func(*args, **kwargs)
        end_time=time()

        processing_time=end_time-start_time
        print(f'the processing time is {processing_time:.2f} secondes')
        return result

    return wrapper

# applying that decorator in a simple function
@measure_time
def simple_function(n):
  """A simple function that calculates the sum of squares of numbers from 1 to n."""
  S= 0
  for i in trange(1, n + 1):
    S+=i**2
  return S

# Example usage
n = 10000000
result = simple_function(n)
www.kaggle.com/competitions/um-game-playing-strength-of-mcts-variants/overview/prizes1010000000....
    def _parse_player_height(self):

        @return_default_value_if_fails(default_value=75)
        def parse(x):
            return 12 * int(x.split('-')[0]) + int(x.split('-')[1])
        return self.df['PlayerHeight'].apply(parse)
....
12

**Pull request merged**